AdblockPlusSafari/AdblockPlus.m

```diff
@@ -17,15 +17,15 @@
 
 #import "AdblockPlus.h"
 
-@import SafariServices;
-
-static NSString *AdblockPlusActivated = @"AdblockPlusActivated";
+NSString *AdblockPlusActivated = @"AdblockPlusActivated";
 static NSString *AdblockPlusEnabled = @"AdblockPlusEnabled";
 static NSString *AdblockPlusAcceptableAdsEnabled = @"AdblockPlusAcceptableAdsEnabled";
+static NSString *AdblockPlusFilterlists = @"AdblockPlusFilterlists";
```

Comment:
I see, I copied this mistake from invisio designs. Unfortunately, it is scattered across multiple places.